fiftyone/operators/store/service.py

```
@@ -0,0 +1,128 @@
+
+import logging
+from typing import Optional, List
+from fiftyone.operators.store.models import StoreDocument, KeyDocument
+
+logger = logging.getLogger(__name__)
+
+
+class ExecutionStoreService:
+    """Service for managing execution store operations."""
+
+    def __init__(self, repo: Optional["ExecutionStoreRepo"] = None):
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)